.github/labeler.yml

```
@@ -72,6 +84,7 @@
   - "**/mapgen/**/*"
   - "**/mapgen_palettes/**/*"
   - "**/overmap/**/*"
+  - "src/overmap*.*"
```

```suggestion
  - "**/overmap**"
```
I'd use this here, we cover these files

> find src/ | rg ".+/overmap"
src/overmapbuffer.h
src/overmap.cpp
src/overmap_connection.cpp
src/overmap_location.cpp
src/overmap.h
src/overmap_noise.h
src/overmap_connection.h
src/overmap_location.h
src/overmap_noise.cpp
src/overmap_types.h
src/overmapbuffer.cpp
src/overmap_ui.h
src/overmap_ui.cpp